src/movement.cpp

```diff
@@ -691,7 +694,7 @@ ReturnValue MoveEvent::EquipItem(MoveEvent* moveEvent, Player* player, Item* ite
 
 	const ItemType& it = Item::items[item->getID()];
 	if (it.transformEquipTo != 0) {
-		Item* newItem = g_game.transformItem(item, it.transformEquipTo);
+		auto newItem = g_game.transformItem(item, it.transformEquipTo);
```

Comment:
Could this be `const auto`
```suggestion
		const auto newItem = g_game.transformItem(item, it.transformEquipTo);
```